ondemand.osc.edu/central_config.yaml

```diff
@@ -0,0 +1,261 @@
+## MOTD_TITLE
+# !!Deprecated!!: Use dashboard.motd_title in the locale file instead
+# Purpose: The title for the MOTD block
+# Type: String | null
+# Default: null
+MOTD_TITLE: null
+## ONDEMAND_VERSION
+# Purpose: ???
+# Type: String
+# Default:
+ONDEMAND_VERSION: null
+## OOD_ANNOUNCEMENT_PATH
+# Purpose: The file system path to the announcements
+# Note: Could be an array; we support usage of an announcements.d we could pull from multiple paths
+# Type: String
+# Default:
+OOD_ANNOUNCEMENT_PATH: null
+## OOD_APP_CATALOG_URL
+# Purpose: The link to the external application catalog page
+# Type: String | null
+# Default: null
+OOD_APP_CATALOG_URL: null
+## OOD_APP_CONFIG_ROOT
+# Purpose: The file system path to the default configuration files for specific apps
+# Note: 
+# Type: String | null
+# Default:
+OOD_APP_CONFIG_ROOT: null
+## OOD_APP_DEVELOPMENT
+# Purpose: Flag whether the developer view should be shown in the Dashboard
+# Note: Considers whether the development app directory exists in the configured path, or is a symlink; see also OOD_DEV_APPS_ROOT
+# Type: Boolean | null
+# Default: false
+OOD_APP_DEVELOPMENT: null
+## OOD_APP_SHARING
+# Purpose: Flag whether the app sharing controls should be shown in the Dashboard
+# Type: Boolean | null
+# Default: false
+OOD_APP_SHARING: false
+## OOD_BC_APP_CONFIG_ROOT
+# Purpose: The file system path to the default configuration files for Batch Connect applications
+# Type: String | null
+# Default: /etc/ood/config/apps
+OOD_BC_APP_CONFIG_ROOT: null
+## OOD_BRAND_BG_COLOR
+# Purpose: Color for the navbar's background (CSS colors)
+# Type: String | null
+# Default: #53565a (Grey)
+OOD_BRAND_BG_COLOR: '#6CACE4'
+## OOD_BRAND_LINK_ACTIVE_BG_COLOR
+# Purpose: Color for the navbar links' background when clicked (CSS colors)
+# Type: String | null
+# Default: #3b3d3f (Darker grey)
+OOD_BRAND_LINK_ACTIVE_BG_COLOR: '#375C84'
+## OOD_DASHBOARD_2FA_URL
+# Purpose: ???
+# TODO: What is the purpose of this? If a user can see this link then they have already authenticated. If they need help with auth, then they won't be able to see this link. ???
+# Type: String | null
+# Default: null
+OOD_DASHBOARD_2FA_URL: null
+## OOD_DASHBOARD_DEV_DOCS_URL
+# Purpose: An external link to documentation on developing applications for OnDemand
+# Type: String | null
+# Default: https://go.osu.edu/ood-app-dev
+OOD_DASHBOARD_DEV_DOCS_URL: null
+## OOD_DASHBOARD_DOCS_URL
+# Purpose: An external link to documentation for OnDemand users
+# Type: String | null
+# Default: null
+OOD_DASHBOARD_DOCS_URL: https://www.osc.edu/ondemand
+## OOD_DASHBOARD_PASSWD_URL
+# Purpose: An external link to your site's password changing service
+# Type: String | null
+# Default: null
+OOD_DASHBOARD_PASSWD_URL: https://my.osc.edu
+## OOD_DASHBOARD_SUPPORT_URL
+# Purpose: An external link to your site's user support
+# Type: String | null
+# Default: null
+OOD_DASHBOARD_SUPPORT_URL: https://www.osc.edu/contact/supercomputing_support
```

Comment:
Instead of having special URLs like these, being able to add arbitrary links to Help would be preferable. Along with arbitrary links to any other navbar menu. One approach:

```yaml
links:
  Jobs:
    - XDMoD: { icon: "https://url/to/xdmod/.png", url: "https://xdmod.osc.edu/" }
  Interactive Apps:
    Desktops:
      - Fast X: { icon: "file:///path/to/fastx.png", url: "https://fastx.osc.edu/" }
  Help:
    - Contact Support: { icon: "question-circle", url: "https://www.osc.edu/contact/supercomputing_support" }
    - Online Documentation: { icon: "info-circle", url: "https://www.osc.edu/ondemand" }
    - Change HPC Password: { icon: "key", url: "https://my.osc.edu/" }
```

* category is the key, and either a list of links, or a subcategory followed by a list of links is allowed
* if subcategory is not provided or "default" is used, then that will work like menus work right now in OnDemand - I believe default links group together at top or bottom of menu separate from any other links that have an explicitly specified subcategory
* default icon string is expected to be a font awesome icon, unless the scheme is in front, and specifying http or https or file

Reply:
Adopted.